pkg/test/utils/kubernetes.go

```diff
@@ -1016,7 +1016,11 @@ func RunCommand(ctx context.Context, namespace string, cmd harness.Command, cwd
 	kuttlENV["NAMESPACE"] = namespace
 	kuttlENV["KUBECONFIG"] = fmt.Sprintf("%s/kubeconfig", actualDir)
 	if kubeconfigOverride != "" {
-		kuttlENV["KUBECONFIG"] = filepath.Join(actualDir, kubeconfigOverride)
+		if filepath.IsAbs(kubeconfigOverride) {
+			kuttlENV["KUBECONFIG"] = kubeconfigOverride
+		} else {
+			kuttlENV["KUBECONFIG"] = filepath.Join(actualDir, kubeconfigOverride)
```

Comment:
this looks off in Go code... it is uncommon to see nested if statements.  It would be better IMO to have a function that takes actualDir and override and returns the proper kubeconfig... or it could just take the override and acquire the actualdir in the function body.   This would simplify this assignment and put the logic in a testable function.  it would also make it more idiomatic go, as the function would just return a value when it is known, resulting code being more left indented.   It's a nit and non-blocker.  it would be better but marginallyl